apps/aecore/src/aec_peer_connection.erl

```diff
@@ -856,10 +856,30 @@ handle_get_generation_rsp(S, {get_generation, From, _TRef}, Msg) ->
     gen_server:reply(From, Res),
     remove_request_fld(S, get_generation).
 
+deserialize_key_block(SKB) ->
```

Comment:
writing a parameterized `deserialize_block` would save some code duplication but introduce a wee bit more complexity... matter of taste really.